dydx/dydxFormatter/dydxFormatter/dydxFormatter.swift

```diff
@@ -369,7 +369,7 @@ public final class dydxFormatter: NSObject, SingletonProtocol {
     /*
      xxxxxx,yyyyy or xxxxx.yyyyy
      */
-    public func raw(number: NSNumber?, size: String?) -> String? {
+    public func raw(number: NSNumber?, size: String?, locale: Locale = Locale.current) -> String? {
```

Comment:
I think we forgot to pass `locale` down.

Reply:
Yup.. Good catch.